### PR TITLE
[terra-menu] Fixed Whitespace issue in Large Menu

### PR DESCRIPTION
**Pull request by adavijit**
### Summary

**What was changed:**

- Added conditional class to the HookshotContent component based on the prop `hookshotPostionFixed`
- Added prop to allow accepting and passing additional props to the Popup component.
- Added and passed `hookshotPostionFixed` prop to set HookshotCoontent position to `fixed`.

**Why it was changed:**

Opening Large Menu and navigated back to main menu from submenu, white space appears at the bottom causing page shift.

### Testing

This change was tested using:

- [x] WDIO
- [x] Jest
- [x] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [ ] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

image

**This PR resolves:**

UXPLATFORM-10055 

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-menu/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
+  * `hookshotPostionFixed` prop to set the position as `fixed` for the menu when used with HookshotContent.
```
Comment (sugan2416) [inline packages/terra-menu/CHANGELOG.md:6]:
```suggestion
  * Fixes white space issue in large menu by setting hookshot position to fixed
```

File: packages/terra-menu/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
```
Comment (sugan2416) [inline packages/terra-menu/CHANGELOG.md:5]:
```suggestion
* Fixed
```

File: packages/terra-popup/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
+  * Added `...customProps` to allow accepting and passing additional props to the Popup component.
```
Comment (sugan2416) [inline packages/terra-popup/CHANGELOG.md:6]:
```suggestion
  * Allow popup to pass through custom props to `terra-hookshot`
```

File: packages/terra-popup/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
```
Comment (sugan2416) [inline packages/terra-popup/CHANGELOG.md:5]:
```suggestion
* Changed
```

File: packages/terra-hookshot/src/HookshotContent.jsx [hunk 5]
```diff
@@ -187,10 +187,14 @@ class HookshotContent extends React.Component {
...
     delete customProps.closePortal;
 
+    const { hookshotPostionFixed } = customProps;
+    delete customProps.hookshotPostionFixed;
+
     return (
       <div
         {...customProps}
-        className={cx(['content', customProps.className])}
+        // eslint-disable-next-line react/forbid-dom-props
```
Comment (sugan2416) [inline packages/terra-hookshot/src/HookshotContent.jsx:196]: This can be removed

File: packages/terra-hookshot/CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Added conditional class to the HookshotContent component based on the prop `hookshotPostionFixed`
```
Comment (sugan2416) [inline packages/terra-hookshot/CHANGELOG.md:6]:
```suggestion
  * Allow hookshot position to be fixed
```

File: packages/terra-hookshot/src/HookshotContent.jsx (hunk 5 above)
Comment (adavijit) [reply packages/terra-hookshot/src/HookshotContent.jsx]: @sugan2416  Removed the line

**Review by sugan2416 — APPROVED** [review]

**Review by saket2403 — APPROVED** [review]

**Pull request merged**